lib/plugins/Label_22_OFF.test.ts

```diff
@@ -0,0 +1,112 @@
+import { MessageDecoder } from '../MessageDecoder';
+import { Label_22_OFF } from './Label_22_OFF';
+
+describe('Label 22 OFF', () => {
+  let plugin: Label_22_OFF;
+
+  beforeEach(() => {
+    const decoder = new MessageDecoder();
+    plugin = new Label_22_OFF(decoder);
+  });
+
+  test('matches qualifiers', () => {
+    expect(plugin.decode).toBeDefined();
+    expect(plugin.name).toBe('label-22-off');
+    expect(plugin.qualifiers).toBeDefined();
+    expect(plugin.qualifiers()).toEqual({
+      labels: ['22'],
+      preambles: ['OFF'],
+    });
+  });
+
+  test('decodes variant 1', () => {
+    const text = 'OFF01YX3661/25251712KIADKPWM171207  92'
+    const decodeResult = plugin.decode({ text: text });
+    expect(decodeResult.decoded).toBe(true);
+    expect(decodeResult.decoder.decodeLevel).toBe('partial');
+    expect(decodeResult.raw.flight_number).toBe('YX3661');
+    expect(decodeResult.raw.departure_day_of_month).toBe(25);
+    expect(decodeResult.raw.arrival_day_of_month).toBe(25);
+    expect(decodeResult.raw.time_of_day).toBe(61920);
+    expect(decodeResult.raw.off_time).toBe(61927);
+    expect(decodeResult.raw.departure_icao).toBe('KIAD');
+    expect(decodeResult.raw.arrival_icao).toBe('KPWM');
+    expect(decodeResult.formatted.items.length).toBe(7);
+    expect(decodeResult.formatted.items[0].label).toBe('Flight Number');
+    expect(decodeResult.formatted.items[0].value).toBe('YX3661');
+    expect(decodeResult.formatted.items[1].label).toBe('Departure Day');
+    expect(decodeResult.formatted.items[1].value).toBe('25');
+    expect(decodeResult.formatted.items[2].label).toBe('Arrival Day');
+    expect(decodeResult.formatted.items[2].value).toBe('25');
+    expect(decodeResult.formatted.items[3].label).toBe('Message Timestamp');
+    expect(decodeResult.formatted.items[3].value).toBe('17:12:00');
```

Comment:
this is a little weird, but what I'm calling the timestamp has minute resolution, but the actual takeoff time has second resolution. Open to ideas

Reply:
I would say that as long as the raw value is still defined with seconds, the formatted can be whatever.